server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/PublishBallotsToAudit.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 12, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @creator Joe Kiniry <kiniry@freeandfair.us>
+ * @description A system to assist in conducting statewide
+ * risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+import static us.freeandfair.corla.asm.ASMEvent.DoSDashboardEvent.PUBLISH_BALLOTS_TO_AUDIT_EVENT;
+
+import spark.Request;
+import spark.Response;
+
+import us.freeandfair.corla.asm.ASMEvent;
+
+/**
+ * Download all ballots to audit for the entire state.
+ * 
+ * @author Joe Kiniry <kiniry@freeandfair.us>
+ * @version 0.0.1
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public class PublishBallotsToAudit extends AbstractDoSDashboardEndpoint {
```

Comment:
Does "Publish" mean "download" as in "you contact this endpoint and it gives you the list of ballots", or does it mean "we set up the internal data structures so that the various dashboard refresh endpoints will have the lists of ballots", or does it mean something else?